deployments/k8s/helm/signalfx-agent/templates/_helpers.tpl

```diff
@@ -55,6 +55,13 @@ Create the name of the secret holding the token
 {{- define "signalfx-agent.secretName" -}}
     {{ default (include "signalfx-agent.fullname" .) .Values.signalFxAccessTokenSecretName }}
 {{- end -}}
+{{/*
+Name of secret holding splunk hec token
+*/}}
+{{- define "signalfx-agent.secretSplunkName" -}}
+    {{ default (include "signalfx-agent.secretName") .Values.splunkTokenSecretName }}
```

Comment:
Don't you need to supply the `.` to the `include` as is done on line 56 above?

Reply:
Fixed